.gitlab-ci/template/include/preferred-wpe-2_30.conf

```diff
@@ -2,5 +2,5 @@ PREFERRED_PROVIDER_virtual/wpebackend = "wpebackend-fdo"
 PREFERRED_PROVIDER_virtual/libwpe = "libwpe"
 PREFERRED_VERSION_cog = "0.8.%"
 PREFERRED_VERSION_wpewebkit = "2.30.%"
-PREFERRED_VERSION_wpebackend-fdo = "1.8.%"
```

Comment:
According with https://wpewebkit.org/release/schedule/ 2.30 is intended to be used only with WPEbackend-fdo 1.8. @aperezdc @zdobersek I know that table is not fully completed. WDYT, should we update the wpewebkit.org docs or keep this test as it is?

Reply:
FWIW I only ended up changing these because the github checks were failing. No real preference whether to mark 1.10 series as compatible even with 2.30.

Reply:
The reason why the WPE WebKit 2.30 + WPEBackend-fdo 1.10.x combination is not in the table is because I have not tried that combination myself when we were preparing for releases last March. I _think_ that it should work fine, if any of you has tried it and reports it as working—I do not have time right now, sorry—, then we can add it in the table at the WPE website ✍🏼

Reply:
Prerequisite for 2.30.x + 1.10.x combo to work is to have it in a building state. So I'll be merging this in as-is.

But yes, 2.30.x + 1.10.x should work fine.